tutorial/lwt/echo_server/config.ml

```diff
@@ -2,7 +2,7 @@
 open Mirage
 
 let main =
-  let packages = [ package "duration"; package ~max:"0.2.0" "randomconv" ] in
+  let packages = [ package "duration"; package "randomconv" ] in
```

Comment:
I guess strictly speaking we should add `~min:"0.2.0"`, but for an example it's probably noise and not that helpful.